### DISCUSSION TITLE
Problemi al riavvio HA

### DISCUSSION THREAD

**Discussion by Osky9** [indomus/forum#539] [HUB: Home Assistant]
Ciao a tutti, dopo aver installato il penultimo aggiornamento HA non si avvia più (HA OS).
Collegandolo al monitor mi esce:
[WARM] Home assistant CLI not starting! Jump into emergency console..

Avete suggerimenti? So già che "non apprezzate" molto questa installazione di HA ma ho preferito partire da qui essendo un neofita (prima installazione di HA fatta a Gennaio di quest'anno).
Il tutto è in funzione su raspberry pi4.
Grazie a tutti.

**Comment by snakuzzo** [comment]
Dovresti postare cosa viene scritto nel log

**Reply by Osky9** [reply]
Come posso fare se non si avvia HA? (scusa la domanda idiota)

**Comment by Osky9** [comment]
OK, ho aperto il file del log e le ultime due righe sono le seguenti:

2021-06-21 14:15:58 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /core/update request Server disconnected
2021-06-21 14:15:58 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /core/update -

**Reply by snakuzzo** [reply]
Problemi di connessione

**Reply by Osky9** [reply]
Cioè? io non riesco ad accedere nemmeno da locale...eppure se mi connetto dal pc via lan entro normalmente nelle cartelle

**Reply by snakuzzo** [reply]
non saprei, mi spiace

**Reply by Osky9** [reply]
mi sto salvando i backup e procederò con una nuova installazione+ripristino
Posso ripristinare un backup di HASSOS su una nuova installazione CORE?

**Reply by snakuzzo** [reply]
no...su Core non hai supervisor e addon. Quindi dovrai sicuramente riportare a mano la dir di conf di HA ed adeguarla dove necessario (anche in base al tipo di installazione che sceglierai...venv o docker).
Inoltre non avendo gli addon dovrai installare altri software come servizi (o nuovi container docker se sceglierai questa via)

**Comment by 50494554524F** [comment]
la versione 6.X è buggata... scarica la 5.13
